libs/core/src/main/java/org/opensearch/ExceptionsHelper.java

```diff
@@ -60,12 +64,18 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import static org.opensearch.OpenSearchException.getExceptionSimpleClassName;
+
 /**
  * Helper class for OpenSearch Exceptions
  *
  * @opensearch.internal
  */
-public final class ExceptionsHelper extends BaseExceptionsHelper {
+public final class ExceptionsHelper {
+    protected static final Logger logger = LogManager.getLogger(ExceptionsHelper.class);
```

Comment:
```suggestion
    private static final Logger logger = LogManager.getLogger(ExceptionsHelper.class);
```

This is final class, no need for protected members